src/pyenphase/envoy.py

```diff
@@ -222,7 +240,8 @@ async def _request(
                 data=orjson.dumps(data),
             )
         else:
-            _LOGGER.debug("Requesting %s with timeout %s", url, self._timeout)
+            if debugon:
+                _LOGGER.debug("Requesting %s with timeout %s", url, self._timeout)
```

Comment:
I don't think we need `debugon` here since there are no arguments being formatted https://docs.python.org/3/howto/logging.html#optimization

Reply:
Ok, thanks for the background link. updated